adsl-client/lang/en

```diff
@@ -20,7 +20,7 @@ index_cdemanddesc=The ADSL client is currently waiting for an on-demand connecti
 index_header=ADSL client configuration options
 index_desc=The settings below apply to any new ADSL connection started by your system. If you change them, the connection must be shut down and re-started for the modifications to take effect.
 index_eth=Ethernet interface
-index_other=Other..
+index_other=Other ..
```

Comment:
Is this label part of a progressive output or text in a button?

Either way, manually changing the formatting like this in the main `lang/en` file shouldn’t be done. If changes are made, they should be applied to all language files.

Jamie has his own reasons for making it the way it is, and neither discussing nor fixing it should be part of the language-related changes.

Reply:
To be clear, in the example of `protocol..` vs `protocol ..` and many other similar cases, this distinction follows the same logic as explain below:

> `protocol..`: The two dots immediately after the word imply an unfinished thought. It suggests continuation.

> `protocol ..`: With the space before the dots, it emphasizes a pause, making it clearer that the word "protocol" is separate from the trailing dots.

The first is mostly used by Jamie in buttons, and the latter in progressive output. It makes sense to me too! Like it or not, this is the way it is, and it shouldn’t be changed when fixing translations.